recipes/postgresql/bld.bat

```diff
@@ -0,0 +1,28 @@
+cd src\tools\msvc
```

Comment:
> So long as you are prepared to do follow up work

Follow up? Yes. Solve it? Unlikely :smile_cat: (But I will call on all our Windows saints: @gillins, @msarahan, and @patricksnape :wink:)